bitnami/kafka/templates/kafka-metrics-deployment.yaml

```diff
@@ -77,43 +77,46 @@ spec:
           {{- end }}
           {{- if .Values.diagnosticMode.enabled }}
           command: {{- include "common.tplvalues.render" (dict "value" .Values.diagnosticMode.command "context" $) | nindent 12 }}
-          {{- else if .Values.metrics.command }}
-          command: {{- include "common.tplvalues.render" (dict "value" .Values.metrics.command "context" $) | nindent 12 }}
+          {{- else if .Values.metrics.kafka.command }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.metrics.kafka.command "context" $) | nindent 12 }}
           {{- else }}
           command:
-            - kafka_exporter
+            - bash
```

Comment:
Why the change?

Reply:
The original form resulted in printing out potential SASL auth environment variables into the log at startup, which I consider to be a fatal flaw. This change allows bash to interpret the environment variables at runtime, allowing it to work nearly identically without printing out the values of the SASL variables.